### ISSUE TITLE
Make WSLg support optional

### ISSUE THREAD

**Issue by nzbr** [nix-community/NixOS-WSL#266]
Everything related to GUI-Support should be separated out into its own module. 
While the module should be enabled by default, it should be possible to disable it in order to reduce the system's footprint for people who don't want/need it

**Comment by kjhoerr** [comment]
Yes please! I can't use WSLg for reasons outside of my control and all the gui support derivations add a lot of time to what should be a pretty flat build.

**Comment by flickerfly** [comment]
Agreed

**Comment by inschrift-spruch-raum** [comment]
pull已经合并了,问题解决了吗
